steps/aws-device-farm-runner/0.0.8/step.yml

```diff
@@ -0,0 +1,231 @@
+title: Amazon Device Farm Runner
+summary: Amazon Device Farm Runner
+description: Deploys app to device farm and starts a test run. [Please see the wiki
+  for setup instructions](https://github.com/peartherapeutics/bitrise-aws-device-farm-runner/wiki).
+website: https://github.com/peartherapeutics/bitrise-aws-device-farm-runner
+source_code_url: https://github.com/peartherapeutics/bitrise-aws-device-farm-runner
+support_url: https://github.com/peartherapeutics/bitrise-aws-device-farm-runner/issues
+published_at: 2020-02-27T10:06:30.466782-08:00
+source:
+  git: https://github.com/peartherapeutics/bitrise-aws-device-farm-runner.git
+  commit: e5080219fcab60ec00cd3e91a13d5f209457a3f0
+host_os_tags:
+- linux
+- osx-10.9
+- osx-10.10
+type_tags:
+- test
+deps:
+  brew:
+  - name: awscli
+    bin_name: aws
+  - name: jq
+    bin_name: jq
+  apt_get:
+  - name: awscli
+    bin_name: aws
+  - name: jq
+    bin_name: jq
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- access_key_id: $AWS_ACCESS_KEY
```

Comment:
We are following a pattern of using envvars as default inputs suggested by another contributor as this makes it easier to test the step. The envvars can be set in the secrets file which can easily be ignored in git, keeping them safe.

We cannot provide a default value for most of these things as they reference AWS credentials and resources specific to the individual user. So I don't see the harm in using a default envvar here since it'd have to be blank otherwise.

Reply:
The reason behind it: the workflow editor highlight required inputs without values, to express the step will not work without setting value for the given input. If you set an env var, which does not have automatic value, as a default value for input the workflow editor will think the required input has value set (if the default env has no value yet).

Other than that there is no reason suggesting env var names, users might have the same value set in a different env var.

Reply:
Makes sense. I think it should still be possible to pass these testing envvars in our step's `bitrise.yml` but keep them out of the `step.yml`. Will look into it.